src/openrct2/drawing/ImageImporter.h

```diff
@@ -41,16 +41,22 @@ namespace OpenRCT2::Drawing
         enum IMPORT_FLAGS
         {
             NONE = 0,
-            KEEP_PALETTE = 1 << 0,
             RLE = 1 << 1,
         };
 
+        enum class PALETTE : uint8_t
+        {
+            OPENRCT2,
+            KEEP_INDICES,
+        };
```

Comment:
This is not our coding style, we usually do
```cpp
enum class MyEnum : uint32_t
{
    First = 1,
    Second,
    Third,
    Count,
}
```

Reply:
I will address that in another PR and also do the other enums for the image importer.